integration/spark2/src/test/scala/org/apache/spark/sql/CarbonAnalysisTest.scala

```diff
@@ -0,0 +1,25 @@
+package org.apache.spark.sql
+
+import org.apache.spark.sql.common.util.Spark2QueryTest
+import org.scalatest.BeforeAndAfterAll
+
+class CarbonAnalysisTest extends Spark2QueryTest with BeforeAndAfterAll {
+
+  override def afterAll(): Unit = {
+    sql("DROP TABLE IF EXISTS table1")
+    sql("DROP TABLE IF EXISTS table2")
+  }
+
+  test("test carbon insert and select table column not equal") {
```

Comment:
Please move the test case class: InsertIntoCarbonTableTestCase， no need to create new one.

Reply:
done